build.sbt.backup

```diff
@@ -0,0 +1,46 @@
+organization := "com.github.biopet"
```

Comment:
This file needs to be removed.
The reason the backup is there is so the dependencies can be easily transferred.